### PR TITLE
Move EDI content 

### PR DESCRIPTION
**Pull request by ckaasi**
- Add TOC partial
- Rewrite sidemenu to use TOC and check for current section
- Add parameter `hidesubmenu` to stop rendering submenu for page
- Add EDI content
  - Move anchor points to separate pages
- Add files folder to hugo

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/20eqWelu](https://trello.com/c/20eqWelu)

File: hugo/content/edi/other-implementation-guides.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+weight: 4
+---
+
+<article class="api">
+  <h1 class="api__title">Other Implementation Guides</h1>
+  <section class="api__section">
+    <h2>Download Documentation</h2>
+    <ul>
+      <li>
+        <a href="{{ site.baseurl }}/files/fortolling.pdf">Customer Handling</a>
```
Comment (halvorsanden) [inline hugo/content/edi/other-implementation-guides.html:17]: `{{ site.baseurl }}` doesn’t work in Hugo. Generally variables don't work in content, only in templates. For this, the related files could go into a pdf folder inside EDI and be relinked from there.

In other cases, where the link is to other pages, it’s usually sufficient to just remove the old `{{ site.baseurl }}` tag. I have done this for the API documentation in the branch I’m working on.

File: hugo/content/edi/status-messages.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+title: Status Messages
+layout: edi
+menu:
+  edi:
+    identifier: "statusMessages"
+    title: "Status Messages"
+weight: 5
+---
+
+<article class="api">
```
Comment (halvorsanden) [inline hugo/content/edi/status-messages.html:11]: The `article` level is provided by the template.

File: hugo/layouts/_default/edi.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  <div class="wrapper">
+    <div class="layout layout--flush">
+      <div class="layout__item palm-one-whole desk-one-quarter">
+        {{ partial "sidemenu.html" . }}
+      </div>
+      <div class="layout__item palm-one-whole desk-three-quarters">
+        <article class="api">
+          {{ $content }}
+        </article>
+        {{ range .Site.Data.api }} {{ end }}
```
Comment (halvorsanden) [inline hugo/layouts/_default/edi.html:14]: No need for this range inside EDI. It only gets the json file for the API

File: hugo/layouts/partials/toc.html [hunk 4]
```diff
@@ -0,0 +1,25 @@
+{{/* require id tag */}}
+{{- $headers := findRE "<h[2-4].* id=\"([^\"]\\S*?)\">(.|\n])+?<\\/h[2-4]>" .context.Content -}}
```
Comment (halvorsanden) [inline hugo/layouts/partials/toc.html:2]: Nice use of `-` which removes the whitespace caused by the templating language. I have gone through most of the other templates and added it (cuts down booking API output from 16000 lines to 9000), but we can take a look at the remaining separately.

**Review by halvorsanden — APPROVED** [review]

**Pull request merged**